subsys/nv_config/src/config_store.c

```diff
@@ -0,0 +1,161 @@
+/*
```

Comment:
why name config_store.c? do we have another file with config_load.c?

Reply:
This is original name of this file form Mynewt.